cli/server/server.go

```diff
@@ -17,13 +17,14 @@ import (
 var version = "dev"
 
 type appConfig struct {
-	Env         string `envconfig:"ENV" default:"production"`
-	Host        string `envconfig:"HOST" default:"0.0.0.0"`
-	Port        int    `envconfig:"PORT" default:"8080"`
-	ServerURL   string `envconfig:"SERVER_URL"`
-	UIDomain    string `envconfig:"UI_DOMAIN"`
-	DBConn      string `envconfig:"DB_CONNECTION" default:"sqlite:///var/code-annotation/internal.db"`
-	ExportsPath string `envconfig:"EXPORTS_PATH" default:"./exports"`
+	Env          string `envconfig:"ENV" default:"production"`
+	Host         string `envconfig:"HOST" default:"0.0.0.0"`
+	Port         int    `envconfig:"PORT" default:"8080"`
+	ServerURL    string `envconfig:"SERVER_URL"`
+	UIDomain     string `envconfig:"UI_DOMAIN"`
+	DBConn       string `envconfig:"DB_CONNECTION" default:"sqlite:///var/code-annotation/internal.db"`
+	ExportsPath  string `envconfig:"EXPORTS_PATH" default:"./exports"`
+	GaTrackingID string `envconfig:"GA_TRACKING_ID" required:"false"`
```

Comment:
We should define this value in the `helm-charts/values.yml`, inside the [`deployment` section](https://github.com/src-d/code-annotation/blob/master/helm-charts/code-annotation/values.yaml#L13). Since we do not know yet its production value, we could use an empty string; it will be replaced with the final `GA_TRACKING_ID` value once it has been provided by https://github.com/src-d/code-annotation/issues/169

The value above should be passed to the env at [helm-charts/templates/deployment.yaml](https://github.com/src-d/code-annotation/blob/master/helm-charts/code-annotation/templates/deployment.yaml#L28)

Reply:
added to helm chart too